zerver/actions/message_send.py

```diff
@@ -1747,7 +1747,7 @@ def check_message(
             # else can sneak past the access check.
             assert sender.bot_type == sender.OUTGOING_WEBHOOK_BOT
 
-        if realm.mandatory_topics and topic_name == "(no topic)":
+        if realm.mandatory_topics and topic_name == "":
```

Comment:
I think we might need to forbid both `""` and `"(no topic)"` during a transitional period. Ideally, we'd be able to merge the whole server part of this PR without any user-facing impact (by just not enabling the client capability), and we don't want to allow the "(no topic)" topic until we're ready to cut over to the feature being fully supported.

Reply:
> I think we might need to forbid both "" and "(no topic)"

Yeah, makes sense. Made the changes.